packages/account/src/Sections/Profile/PhoneVerification/otp-verification.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
```

Comment:
If this component is not shared with other package, may be only required functions or components can be imported instead of the whole react.